calyx-opt/src/passes/top_down_static_timing/tdst.rs

```diff
@@ -364,6 +361,11 @@ impl Schedule<'_, '_> {
             ir::Control::Par(par) => {
                 self.par_calculate_states(par, preds)
             }
+            ir::Control::Static(_) => {
+                unimplemented!(
+                "we don't know what to do with `static` statements yet."
```

Comment:
nit: probably just make this "TDST does not currently support `static` statements"